### PR TITLE
DirectLink 1.0

### PR DESCRIPTION
**Pull request by fawazar94**
- Name: DirectLink
- Author: Fawaz Abdul rahman
- Repo: https://github.com/fawazar94/directlink
- Version: 1.0
- NVDA compatibility: 2019.3 or later

### CODE REVIEW

File: get.php [hunk 1]
```diff
@@ -146,6 +146,7 @@
 	"winwizard" => "https://github.com/lukaszgo1/winWizard/releases/download/V5.0.3/winWizard-5.0.3.nvda-addon",
 	"wordnav" => "https://github.com/mltony/nvda-word-nav/releases/download/v1.2/wordNav-1.2.nvda-addon",
 	"zoom" => "zoomEnhancements-1.0.nvda-addon",
+	"directlink" => "https://github.com/fawazar94/DirectLink/releases/download/1.0/directLink-1.0.nvda-addon",
```
Comment (josephsl) [inline get.php:149]: Could you mind repositioning the new entry so everything can be sorted? Thanks.

Comment (fawazar94) [reply get.php]: i've repositioned it to be in d, before dw.

**Comment by josephsl** [comment]
Hi, I’ll take a look at this add-on and will do something about it this afternoon my time. Thanks.

**Comment by nvdaes** [comment]
@josephsl, OK, feel free  to review this. I won't do since this won't be needed. Anyway, just for coordination, I remember this message sent by me to the add-ons mailing list. My intention was to review this this weekend.

El 22/06/2021 a las 11:46, Noelia Ruiz escribió:
> Hello, I'll be happy approving a pull request and requesting the
> needed review from Reef (NV Access) the next weekend.
> I'll try to avoid approving new add-ons to be included on the website
> before a week since the author has requested a review, so we all, or
> most people, has a chance to familiarize with the add-on, provide
> feedback and more people can participate in processes.
> Cheers

https://nvda-addons.groups.io/g/nvda-addons/topic/83646299

**Comment by josephsl** [comment]
Hi, in this case, go ahead and review this one. Thanks.

**Comment by nvdaes** [comment]
OK, I'll do it. Though my intention was just to point out the thread
for coordination, of course no problem if you want to review whatever.
Cheersx

2021-06-24 21:23 GMT+02:00, Joseph Lee ***@***.***>:

**Review by nvdaes — COMMENTED** [review]
@fawazar94 
- You provide a link to download the add-on from your GitHub account. Then, is there any reason for storing the binary file in addonFiles folder? Otherwise, can you remove it, in case this binary (.nvda-addon file) is not needed?
Apart from this, I see that message in line 103 of the Python file in the global plugin doesn't have comment for translators. You don't need to provide it if you don't want, or you can do it in a later version. This doesn't affect this review.

Also, when this PR can be merged, the documentation needs to be updated in a SVN repository so it appears on the website. If you want to do it yourself, please request an invitation in the NVDA's translations mailing list. Otherwise, let me know and I'll update the documentation.

Cheers

**Comment by fawazar94** [comment]
Hi, 
The file is not needed, I just thought this is the right way, so I just added the file just incase.
 it is the very same file which is on the link. 
The message is clear, the comment i'll add in a later version, I missed it when I was adding the comments apparently.
So I remove the file and push again?
for the documentation, yes please update in the repo, I am not fully aware of the translation workflow.
I can request in the mailing list no problem, but if you can do it that would be nice.
thanks.

**Comment by nvdaes** [comment]
> 
> 
> Hi,
> The file is not needed, I just thought this is the right way, so I just added the file just incase.
> it is the very same file which is on the link.
> The message is clear, the comment i'll add in a later version, I missed it when I was adding the comments apparently.
> So I remove the file and push again?

Yes please. Then, I'll approve this PR.

> for the documentation, yes please update in the repo, I am not fully aware of the translation workflow.
> I can request in the mailing list no problem, but if you can do it that would be nice.
> thanks.


OK, I'll update the documentation this time.

If you want to get your add-on registered in the system to receive translations for the gui, please request it on the add-ons mailing list as mentioned in 

https://github.com/nvaccess/mrconfig

**Review by nvdaes — APPROVED** [review]
Thanks.
Also, for the next version, you may remove a leading space in the manifest description, placed after """. This is not important, just mentioning it for your interest.

**Comment by fawazar94** [comment]
hi, 
I've added a branch named 'stable' to my repo, 
that is all required from me? now I send the mail to the translation list?
thanks.

**Comment by nvdaes** [comment]
Hello, I'll fork your repo in nvdaaddons organization and create an issue in mrconfig repo requesting the registration in the translations system. According to instructions this should be requested in the add-ons mailing list, but I think this is just an irientative: the important think is that a member of the nvdaaddons team is aware, and also NV Access.

You can send an email to the translations mailing list if you want to update the documentation on the website yourself.

**Pull request merged**